src/app/root-content/dashboard/widgets/widgets.component.scss

```diff
@@ -0,0 +1,77 @@
+:host {
+	display: block;
+	.ui.cards .ui.card.widget-container {
+		box-shadow: none;
+	}
+
+	.ui.fluid.card {
+		font-size: .95em;
+		box-shadow: none;
+	}
+
+	.ui.card.error {
+		/deep/ .ui.table, /deep/ .ui.header {
+			color: gray;
+		}
+		/deep/ .widget-text {
+			flex: auto;
+			color: gray;
+
+		}
+
+		/deep/ .widget-header {
+			background-color: #bcbec0;
+			padding: .5em .5em;
+		}
+	}
+
+	.ui.cards {
+		.ui.card {
+			font-size: .95em;
+			width: 245px;
+
+			/deep/ .widget-header {
+				background-color: orange;
+				padding: .5em .5em;
+			}
+
+			/deep/ .widget-button {
+				cursor: pointer;
+			}
+
+			/deep/ .widget-body {
+				display: flex;
+				flex-direction: row;
+			}
+
+			/deep/ .widget-icon {
+				height: 60px;
+			}
+
+			/deep/ .widget-text {
+				flex: auto;
+				color: black;
+				line-height: normal;
+				font-size: 50px;
+			}
+		}
+
+		.ui.card.error {
+			/deep/ .widget-text {
+				flex: auto;
+				color: gray;
+				line-height: normal;
+				font-size: 50px;
+			}
+			/deep/ .widget-header {
+				background-color: #bcbec0;
+				padding: .5em .5em;
+			}
+		}
+	}
+
+	/deep/ .ui.card.hidden {
+		display: none;
+	}
+
+}
```

Comment:
I agree with Jimena. If the rules apply to all widgets they should be in a common ancestor (like the container). I understand your point Alvaro but I can't see how we would style them without repeating code... well... maybe SCSS `@import` but that would still mean the rules are going to be repeated on the distribution files. For now, leaving them in the container should be good enough.